Bookish/Bookish.Web/Controllers/HomeController.cs

```diff
@@ -12,18 +13,32 @@ namespace Bookish.Web.Controllers
     public class HomeController : Controller
     {
         private readonly ILogger<HomeController> _logger;
+        private readonly LibraryService libraryService;
 
         public HomeController(ILogger<HomeController> logger)
         {
             _logger = logger;
+            libraryService = new LibraryService();
```

Comment:
This should be passed in by dependency injection. To do that:
1. Create an interface for the class, named `ILibraryService`. This can be defined in the same file as the library service class and should just contain the method signatures for the public methods in your library service class. 
2. Make the library service class implement the interface.
3. Make this constructor take an `ILibraryService` as a parameter.
4. In `ConfigureServices` in `Startup.cs` call 
```
services.AddScoped<ILibraryService, LibraryService>();
```

🙂